### PR TITLE
[Relationship] Add  relationship between Aws-Autoscaling and IAM role

### PR DESCRIPTION
**Pull request by deepak4566**
**Notes for Reviewers**

This PR fixes #

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
I'm bumping this PR to see if the recently RUBY_VERSION env var in Netlify takes effect and build the docs correctly.

**Comment by RipulHandoo** [comment]
@deepak4566 could you please provide a screenshot or video demonstrating the functionality of this relationship?

File: server/meshmodel/aws-applicationautoscaling-controller/relationships/edge_permission.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    "allow": {
+
+      "from": [
+
+        {
+          "kind": "AdoptedResource",
+          "model": "aws-applicationautoscaling-controller",
+          "match": {
+
+            "self": [],
```
Comment (RipulHandoo) [inline server/meshmodel/aws-applicationautoscaling-controller/relationships/edge_permission.json:32]: Please add the appropriate path in the 'self' field under 'match'.

**Review by RipulHandoo — CHANGES_REQUESTED** [review]

File: server/meshmodel/aws-applicationautoscaling-controller/relationships/edge_permission.json (hunk 1 above)
Comment (leecalcote) [reply server/meshmodel/aws-applicationautoscaling-controller/relationships/edge_permission.json:32]: @deepak4566 have you addressed this feedback?

**Pull request closed without merging**